qa/acceptance-tests/src/test/java/io/camunda/it/client/UsageMetricsTest.java

```diff
@@ -80,25 +84,38 @@ static void setup(@Authenticated(ADMIN) final CamundaClient adminClient)
       throws InterruptedException {
     createTenant(adminClient, TENANT_A);
     createTenant(adminClient, TENANT_B);
+    createTenant(adminClient, TENANT_C);
     assignUserToTenant(adminClient, ADMIN, TENANT_A);
     assignUserToTenant(adminClient, ADMIN, TENANT_B);
+    assignUserToTenant(adminClient, ADMIN, TENANT_C);
 
     // Create PI & wait for metrics to be exported
```

Comment:
🙃 It's not just PI anymore.

```suggestion
    // Create first batch of metrics
```